source/connection-monitoring-and-pooling/tests/README.rst

```diff
@@ -36,7 +36,23 @@ Unit Test Format:
 
 All Unit Tests have some of the following fields:
 
-- ``poolOptions``: if present, connection pool options to use when creating a pool
+- ``poolOptions``: If present, connection pool options to use when creating a pool;
+  both `standard ConnectionPoolOptions <https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#connection-pool-options-1>`__
+  and the following test-specific options are allowed:
+
+  - ``backgroundThreadIntervalMS``: A desired minimal time interval between the end of a
```

Comment:
@BorisDog wants wording that discourages implementations from choosing an interval much smaller than the desired one. This is an attempt to prevent implementations from being too inefficient by deciding to use a too short interval or no interval at all (with DRIVERS-1465 the interval will affect not only tests). Given that the whole CMAP spec is purely about efficiency, this is not unreasonable.

I am fine with both "minimal desired" and just "desired" wordings.

Reply:
Maybe we ought to drop "desired" then too? There shouldn't be any choice being made here by the drivers--if an interval is specified they must adhere to it as exactly as they reasonably can (barring any noise from waiting on lock acquisitions and what not).

Reply:
At this point I am fine with any wording once a consensus is reached between you and Boris. Currently it appears to me that there is no consensus. @BorisDog, are you fine with just saying _"A time interval"_ without any further clarification on what is expected from the implementation?

Reply:
Done.